src/menus/cheat_menu.cpp

```diff
@@ -104,6 +106,13 @@ namespace Cheats {
                         tp_clawshot.pull_rate = 500.0f;
                         break;
                     }
+                    case DoorStorage: {
+                        if (tp_gameInfo.link_collision_ptr != nullptr) {
+                            tp_gameInfo.link_collision_ptr->chest_collision = 0xE4;
+                            tp_gameInfo.link_collision_ptr->door_collision = 0x40;
+                        }
+                        break;
+					}
```

Comment:
There's a large mix of tabs and spaces throughout these changed files. Please make sure all indentations are spaces going forward.